src/integrationTest/java/org/opensearch/test/framework/certificate/TestCertificates.java

```diff
@@ -142,12 +148,12 @@ private void isCorrectNodeNumber(int node) {
         }
     }
 
-    private CertificateData createNodeCertificate(Integer node) {
+    private CertificateData createNodeCertificate(Integer node, String nodeSanIp) {
         final var subject = String.format(NODE_SUBJECT_PATTERN, node);
         String domain = String.format("node-%d.example.com", node);
         CertificateMetadata metadata = CertificateMetadata.basicMetadata(subject, CERTIFICATE_VALIDITY_DAYS)
             .withKeyUsage(false, DIGITAL_SIGNATURE, NON_REPUDIATION, KEY_ENCIPHERMENT, CLIENT_AUTH, SERVER_AUTH)
-            .withSubjectAlternativeName("1.2.3.4.5.5", List.of(domain, "localhost"), "127.0.0.1");
+            .withSubjectAlternativeName("1.2.3.4.5.5", List.of(domain, "localhost"), nodeSanIp);
```

Comment:
I know its not introduced in this PR, but at some point I think we need to remove this OID from the default value [here](https://github.com/opensearch-project/security/blob/b8c76eed6f409ac76235c9594493665396dd1047/src/main/java/org/opensearch/security/transport/OIDClusterRequestEvaluator.java#L47).